### PR TITLE
TypeScript Definition: Make `isSame` accept `quarter` as unit with `quarterOfYear` imported

### PR DESCRIPTION
**Pull request by Cryrivers**
Make `isSame` accept `quarter` as unit

### CODE REVIEW

**Comment by iamkun** [comment]
Thanks. You are right. And we should update isAfter isBefore as well. Plus, add a blank line, please.

**Comment by Cryrivers** [comment]
Fixed.

File: types/plugin/quarterOfYear.d.ts [hunk 1]
```diff
@@ -16,5 +16,11 @@ declare module 'dayjs' {
     startOf(unit: QUnitType): Dayjs
 
     endOf(unit: QUnitType): Dayjs
+
+    isSame(date: dayjs.ConfigType, unit?: QUnitType): Dayjs
```
Comment (iamkun) [inline types/plugin/quarterOfYear.d.ts]: Would you import { ConfigType } at top of the file, please?

Comment (Cryrivers) [reply types/plugin/quarterOfYear.d.ts]: Fixed. Thanks

File: types/plugin/quarterOfYear.d.ts [hunk 2]
```diff
@@ -16,5 +16,11 @@ declare module 'dayjs' {
     startOf(unit: QUnitType): Dayjs
 
     endOf(unit: QUnitType): Dayjs
+
+    isSame(date: ConfigType, unit?: QUnitType): Dayjs
```
Comment (iamkun) [inline types/plugin/quarterOfYear.d.ts]: should return a boolean

Comment (Cryrivers) [reply types/plugin/quarterOfYear.d.ts]: Fixed.

**Pull request merged**